packages/block-editor/src/components/writing-mode-control/stories/index.story.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { isRTL } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import WritingModeControl from '../';
+
+/**
+ * WritingModeControl component allows selecting writing mode.
+ */
```

Comment:
The description is already defined [here](https://github.com/WordPress/gutenberg/pull/67343/files#diff-c836fbdea3b54e55e637d06ffc45758c46f13ac4b2d99a38bd2ca0fdaf5a6bedR22), so this comment is unnecessary.